gen/sam/sam_sa15.py

```diff
@@ -15,13 +15,18 @@ class SA15Generator(AirDefenseGroupGenerator):
     price = 55
 
     def generate(self):
-        self.add_unit(
-            AirDefence.Tor_9A331,
-            "ADS",
-            self.position.x,
-            self.position.y,
-            self.heading,
+        num_launchers = 2
```

Comment:
http://www.ausairpower.net/APA-9K331-Tor.html#mozTocId178145 suggests this should be 4

Reply:
So my initial recommendation was to reduce it to 2 which would act as 'half-batteries' so they could be spread out to cover more ground. But reviewing the info I have, I can't really find any details on how a unit as small as a battery would operate. The implication of this diagram I posted in discord (link below) would imply that the battery operates as a fairly cohesive unit so maybe it should actually be 4 per generator.

Given this is all the info we have to go on so far, we can either go with 4 as our best guess at reality or go with 2 if we want to make it easier on players. Which ever we decide on here should also apply to the SA-8 generator.

https://cdn.discordapp.com/attachments/772053150407852044/787374086959398973/unknown.png

Reply:
I made the change not to be realistic, but to be playable. I would leave it at 2.